tests/shared/common.go

```diff
@@ -27,10 +27,10 @@ import (
 
 // ConfigVersions used for all unit tests
 const (
-	PFlexConfigVersion       string = "v2.8.0"
```

Comment:
I would only change the PFlex version, I think changing the others is causing the unit tests to fail. ConfigVersion, UpgradeConfigVersion, and JumpUpgradeConfigVersion are used by PScale tests, not PFlex tests.

Reply:
We need to change these also ConfigVersion, UpgradeConfigVersion, and JumpUpgradeConfigVersion. I verfied powerflex unit tests also depend on the above variables.  The unit tests are passing now